driver/pelican/main.go

```diff
@@ -40,9 +47,9 @@ func main() {
 	password := params.MustString("password")
 	sitename := params.MustString("sitename")
 
-	pelicans, err := DiscoverPelicans(username, password, sitename)
+	pelicans, err := storage.ReadPelicans(username, password, sitename)
```

Comment:
How does discovery work now? Are we still discovering the set of thermostats when the driver boots?

Reply:
No, the driver queries the remote database to learn about the thermostats installed at the relevant site and their true number of heating and cooling stages.

The database is populated by manually running the tool I put in the `discovery` directory, which uses the old discovery mechanism via the API.

Reply:
Ok, thanks; that makes sense